src/db/migrations/26-add-crdt-messages.ts

```diff
@@ -0,0 +1,47 @@
+import { Knex } from 'knex';
+
+import { CrdtOpetation } from '../../types/message';
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.createTable(Table.seeds, table => {
```

Comment:
should we add an explicit auto-incrementing id field here to use as a primary key?